foxy/distribution.yaml

```diff
@@ -5482,7 +5482,7 @@ repositories:
       tags:
         release: release/foxy/{package}/{version}
       url: https://github.com/septentrio-users/septentrio_gnss_driver_ros2-release.git
-      version: 1.2.0-2
+      version: 1.2.0-1
```

Comment:
This doesn't look correct. It is decrementing the revision number; can you confirm?

Reply:
Thank you for pointing that out. We have opened https://github.com/ros/rosdistro/pull/33088 instead.